index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="styles.css" rel="stylesheet" />
+    <title>Landing Page</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <a href="#" class="logo">
+          <img src="./images/logo.jpg" alt="Learning bulb" />
+        </a>
+        <ul class="nav-icons">
+          <li>
+            <a href="#" class="links">Home</a>
+          </li>
+          <li>
+            <a href="#" class="links">Features </a>
+          </li>
+          <li>
+            <a href="#" class="links">Popular</a>
+          </li>
+          <li>
+            <a href="#" class="links">Contact Us</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="headline">
+        <div class="headline-text">
+          <h1>
+            Play, Learn and<br />
+            Grow Together
+          </h1>
+          <p>
+            There are many variations of passages of Lorem available, but the
+            majority have suffered alteration in some form, by injected humour,
+            <br />
+            or randomised words which dont look even slightly believable.<br />
+           Lorem ipsum dolor sit amet consectetur
+            adipisicing elit. <br />Doloremque quo tenetur odit recusandae
+            soluta porro ab officiis molestiae enim atque.
+          </p>
+          <button class="btn sign-up-btn" type="button">Sign up</button>
+        </div>
+
+        <div class="headline-image">
+          <img src="./images/Headlineimg.jpg" alt="Students Learning" />
+        </div>
+      </section>
+      <section class="info">
+        <div class="teachers">
+          <h2>Our Teachers</h2>
+          <ul class="teachers-img">
+            <li>
+              <div class="img-teacher">
+                <img
+                  src="./images/female teacher cynthia.jpg"
```

Comment:
- [ ] All your images are breaking because they are found in the images folder. 
    - Your headline image and logo are misspelled 
    - others are not found. 
    - please ensure there are no spaces in the image names. You can use underscore or dash.
- [ ] Kindly fix it. 

![image](https://user-images.githubusercontent.com/84156714/193518841-08d0ebb9-c553-431d-8285-89ebb6811993.png)